stingray/pulse/pulsar.py

```diff
@@ -771,9 +771,20 @@ def get_orbital_correction_from_ephemeris_file(mjdstart, mjdstop, parfile,
     toalist = _load_and_prepare_TOAs(mjds, ephem=ephem)
     m = get_model(parfile)
     delays = m.delay(toalist)
-    correction_mjd = \
+
+    correction_mjd_rough = \
         interp1d(mjds,
-                 (toalist.table['tdbld'] * units.d - delays).to(units.d).value)
+                 (toalist.table['tdbld'] * units.d - delays).to(units.d).value,
+                  fill_value="extrapolate")
+
+    def correction_mjd(mjds):
```

Comment:
Is this a user-facing function? This should probably have a docstring? :)

Reply:
Ehm (blush)
Yep I'll add it